projects/client/RabbitMQ.Client/src/client/properties/AssemblyInfo.cs

```diff
@@ -50,6 +49,4 @@
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 
-[assembly: AssemblyVersion("4.0.0")]
-[assembly: AssemblyInformationalVersion("4.0.0-pre.1")]
```

Comment:
If the AssemblyInformationalVersion attribute isn't hardcoded, then it uses the version value from the project.json file. 

So, if you're keeping that version number up to date, then the nuget package output from `dotnet pack` will have the correct version, and the informational version will match as well.

Keeping the AssemblyVersion attribute as it is is desirable also, so that binding redirects won't be needed between minor/patch releases.